CharlieBackend.Api.UnitTes/ScheduleServiceTests.cs

```diff
@@ -409,5 +414,438 @@ public async Task CreateScheduleAsync_NullRequest_ShouldReturnValidationError()
+                        MentorId = existentMentorId,
+                        EventStart = DateTime.Parse("Jan 1, 2021"),
+                        EventFinish = DateTime.Parse("Feb 1, 2021"),
+                        EventOccuranceId = 1
+                } },
+                Id = 0
+            };
+            var startDate = DateTime.Parse("Jan 1, 2021");
+            var finishDate = DateTime.Parse("Feb 1, 2021");
+
+            _eventOccuranceRepositoryMock.Setup(x => x.IsEntityExistAsync(existingId)).ReturnsAsync(true);
+            _eventOccuranceRepositoryMock.Setup(x => x.GetByIdAsync(existingId)).ReturnsAsync(validEventOccurrence);
+
+            var scheduleService = new ScheduleService(_unitOfWorkMock.Object, _mapper, _scheduledEventFactory);
+
+            //Act
+            var result = await scheduleService.DeleteScheduleByIdAsync(existingId, startDate, finishDate);
+
+            //Assert
+            result.Should().NotBeNull();
+            result.Data.Should().BeEquivalentTo(expected);
+        }
+
+        [Fact]
+        public async Task UpdateScheduledEventByID_NonExistingScheduledEventId_ShouldReturnValidationError()
+        {
+            //Arrange
+            UpdateScheduledEventDto request = new UpdateScheduledEventDto();
+
+            _scheduleRepositoryMock.Setup(x => x.IsEntityExistAsync(nonExistentId)).ReturnsAsync(false);
+
+            var scheduleService = new ScheduleService(_unitOfWorkMock.Object, _mapper, _scheduledEventFactory);
+
+            //Act
+            var result = await scheduleService.UpdateScheduledEventByID(nonExistentId, request);
+
+            //Assert
+            result.Should().NotBeNull();
+            result.Error.Code.Should().BeEquivalentTo(ErrorCode.ValidationError);
+        }
+
+        [Fact]
+        public async Task UpdateScheduledEventByID_ExistingScheduledEventIdNotValidGroupId_ShouldReturnValidationError()
+        {
+            //Arrange
+            var request = new UpdateScheduledEventDto
+            {
+                StudentGroupId = nonExistentId
+            };
+
+            _studentGroupRepositoryMock.Setup(x => x.IsEntityExistAsync(nonExistentId)).ReturnsAsync(false);
+
+            _scheduleRepositoryMock.Setup(x => x.IsEntityExistAsync(existingId)).ReturnsAsync(true);
+
+            Initialize(validScheduleDTO);
+
+            var scheduleService = new ScheduleService(_unitOfWorkMock.Object, _mapper, _scheduledEventFactory);
+
+            //Act
+            var result = await scheduleService.UpdateScheduledEventByID(existingId, request);
+
+            //Assert
+            result.Should().NotBeNull();
+            result.Error.Code.Should().BeEquivalentTo(ErrorCode.ValidationError);
+        }
+
+        [Fact]
+        public async Task UpdateScheduledEventByID_ExistingScheduledEventIdNotValidMentorId_ShouldReturnValidationError()
+        {
+            //Arrange
+            var request = new UpdateScheduledEventDto
```

Comment:
Why call variable for DTO object a `request` instead of `updateScheduleEventDto`?

Reply:
Fixed